runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaClientConnectionPool.java

```diff
@@ -909,6 +951,9 @@ private void onSignal(
             case SIGNAL_STREAM_INITIAL_RESET:
                 doStreamReset(traceId);
                 break;
+            case SIGNAL_STREAM_REPLY_WINDOW:
+                doStreamWindow(traceId);
```

Comment:
This method is not a `doXXX` method as it doesn't write a frame for stream.

Please rename `doStreamWindow` to `onStreamSignalReplyWindow`.

Also, given that the signals are used to defer something that happened, it might be better to name them for where they originated, rather than how you want them to react.

For example, `SIGNAL_STREAM_BEGIN` (triggered from `onStreamBegin`) and then the reaction can either be `doStreamBegin` directly or `doStreamBegin` called from `onStreamSignalBegin`.